lang/Deutsch (German)

```diff
@@ -1,5 +1,5 @@
 ﻿[WoR lang file]
-WindowTitle=Windows on Raspberry
+WindowTitle=Windows auf dem Raspberry
```

Comment:
This string should not be modified since that's the name of the application.
I'll remove it anyway in the next major version.

Reply:
So I should change this and then you mege?

Reply:
Yes.

Reply:
I did it